app/assets/v2/css/grants/fund.css

```diff
@@ -146,22 +153,101 @@
   bottom: -1.2rem;
 }
 
-.nav{
+.nav {
   text-align: center;
 }
-.nav-link{
+
+.nav-link {
   border-right: 1px solid black;
   border-top: 1px solid black;
   border-left: 1px solid black;
   margin-right: 10px;
 }
-.nav-link:visited{
+
+.nav-link:visited {
   color: black;
 }
-.nav-item.active{
+
+.nav-item.active {
   text-decoration: underline;
 }
 
+.clr_estimate {
+  background-color: white;
+  border-radius: 5px;
+  border: 2px solid #10cd7a;
+  text-align: left;
+  width: 250px;
+  padding: 1rem 15px;
+  position: absolute;
+  top: 0px;
+  left: 0px;
+}
+
+.clr_estimate.extraleft {
+  left: -270px;
+}
+
+.clr_estimate:after {
+  content: '';
+  position: absolute;
+  right: 0;
+  top: 30%;
+  width: 0;
+  height: 0;
+  border: 20px solid transparent;
+  border-left-color: #10cd7a;
+  border-right: 0;
+  border-bottom: 0;
+  margin-top: -10px;
+  margin-right: -20px;
+}
+
+.clr_estimate img {
+  float: right;
+  width: 50px;
+}
+
+.clr_estimate h3 {
+  text-align: left;
+  color: white;
+  font-size: 12px;
+  text-transform: uppercase;
+  width: 150px;
+}
+
+.clr_estimate h3 span {
+  background-color: #10cd7a;
+  border-radius: 3px;
+  padding: 4px 6px;
+
+}
+
+.clr_estimate h4 {
+  padding-top: 0px;
+  text-align: left;
+  color: #10cd7a;
+  font-size: 10px;
+  font-weight: bold;
+}
+
+.clr_estimate h4 span {
+  font-size: 16px;
+}
+
+.clr_estimate p {
+  text-align: left;
+  color: #10cd7a;
+  font-size: 12px;
+  color: #959595;
+}
+
+#phantom_fund .description i {
```

Comment:
any I inside that description should follow this rule?

Reply:
yup